freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserSyncForStackService.java

```diff
@@ -37,6 +39,8 @@ public class UserSyncForStackService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(UserSyncForStackService.class);
 
+    private static final int LARGE_GROUP_LOG_THRESHOLD = 500;
```

Comment:
I don't know if it's worth to make it configurable from app yaml

Reply:
I'll make it configurable. I think it'll be useful to have a definition of a "large group" if we need to condition other behaviors.